File: Java/Events/GetAvailablePerformanceTicketsResult.java
package LTD_API;

import java.util.List;

public class GetAvailablePerformanceTicketsResult
{
    private int performanceId;
    private int eventId;
    private List<TicketAreaInfo> ticketAreas;
    
    public int getPerformanceId()
    {
        return performanceId;
    }
    public void setPerformanceId(int performanceId)
    {
        this.performanceId = performanceId;
    }

    public int getEventId()
    {
        return eventId;
    }
    public void setEventId(int eventId)
    {
        this.eventId = eventId;
    }

    public List<TicketAreaInfo> getTicketAreas()
    {
        return ticketAreas;
    }
    public void setTicketAreas(List<TicketAreaInfo> ticketAreas)
    {
        this.ticketAreas = ticketAreas;
    }
}

File: PHP/Events/GetAvailablePerformanceTicketsResult.php
<?php
namespace LTD_API;

class GetAvailablePerformanceTicketsResult
{
    private $performanceId;
    private $eventId;
    //array of TicketAreaInfo
    private $ticketAreas;
    
    public function getPerformanceId()
    {
        return $this->performanceId;
    }
    public function setPerformanceId($performanceId)
    {
        $this->performanceId = $performanceId;
    }

    public function getEventId()
    {
        return $this->eventId;
    }
    public function setEventId($eventId)
    {
        $this->eventId = $eventId;
    }

    public function getTicketAreas()
    {
        return $this->ticketAreas;
    }
    public function setTicketAreas($ticketAreas)
    {
        $this->ticketAreas = $ticketAreas;
    }
}

